<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CovidTestForm</class>
 <widget class="QWidget" name="CovidTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="lbl_topBar">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>-15</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/images/topBar.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_backToAdminHome">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/backArrow2.png);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_QRRequests">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font-size:28px;
font-family:arial;</string>
     </property>
     <property name="text">
      <string>Add Test Result</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>371</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="lbl_email">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>46</y>
       <width>331</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Date of test</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_testDate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>331</width>
       <height>34</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>YY/MM/DD</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtn_testPos">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Positive</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtn_testNeg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Negative</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>SUBMIT</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>331</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="checkBox_affirmAccuracy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>I affirm the accuracy of these results</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainText_NoEdit_resultsAccuracy">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>291</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>These self-tested results are accurate to the best of my knowledge.
I understand that a false declaration can incur a fine of no more than NZ$5,000.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
